src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,59 @@
+package racingcar.model;
+
+import racingcar.utils.RandomNumberGenerator;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void checkForwardConditionAndGo() {
+        for (Car car : cars) {
+            car.increaseDistance();
+        }
+    }
+
+    public List<Car> getCars() {
+        return Collections.unmodifiableList(this.cars);
+    }
+
+    public String getWinner() {
+        int maxDistance = getMaxDistance();
+        return cars.stream().filter(car -> car.hasMaxDistance(maxDistance)).map(Car::getName).collect(Collectors.joining(", "));
```

Comment:
stream을 사용할땐 줄바꿈을 해주시면 조금 더 가독성이 좋아져요! :) 

```suggestion
       return cars.stream()
                .filter(car -> car.hasMaxDistance(maxDistance))
                .map(Car::getName)
                .collect(Collectors.joining(", "));
```